tests/integrations/raftstore/mod.rs

```diff
@@ -32,4 +32,5 @@ mod test_transfer_leader;
 mod test_transport;
 mod test_unsafe_recovery;
 mod test_update_region_size;
+mod test_v1_v2_mixed;
```

Comment:
There're already some mixed snapshot tests in test_snap.

Reply:
Renamed the test.

Reply:
Actually I meant the placement should be consistent. I prefer to move v1-v2 tests from test_snap to test_v1_v2_mixed.

Reply:
Moved.